allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/interceptors/testrun/VideoRecordingTestInterceptor.kt

```diff
@@ -26,16 +25,10 @@ class VideoRecordingTestInterceptor(
 
     override fun onTestFinished(testInfo: TestInfo, success: Boolean) {
         videos.saveAndApply {
-
             val stubFile = resourceFilesProvider.provideStubVideoFile(this)
             stubFile.attachVideoToAllureReport()
             val uuid = Allure.lifecycle.getCurrentTestCase() ?: ""
-            Allure.lifecycle.updateTestCase {
-                it.status = if (success) Status.PASSED else Status.FAILED
-            }
-            Allure.lifecycle.stopTestCase(uuid)
-            Allure.lifecycle.writeTestCase(uuid)
-            videosHolder.rememberAttachedVideo(stubFile = stubFile, actualFile = this)
+            videosHolder.rememberAttachedVideo(stubFile = stubFile, actualFile = this, uuid = uuid)
```

Comment:
Allure.lifecycle.getCurrentTestCase() returns test cae uuid. If you meant to save video uuid to videosHolder I suppose it's not what you want. If you wanted to save test case uuid into AttachedVideo model? please rename uuid field to something like `testCaseUuid` to make it clear